test/util/data/tapyrus-util-test.json

```diff
@@ -733,5 +733,59 @@
     ],
     "check":"check_generateKey",
     "description": "Ensure generatekey is successful in dev mode"
+  },
+  {
+    "exec": "./tapyrus-genesis",
+    "args": [
+      "-time=1563342688",
+      "-signblockpubkey=04473757a955a23f75379820f3071abf5b3343b78eb54e52373d06259ffa6c550bd0acc38ff1b182a5de0c795aa7109e8348ca5436d65a267ef134af595ec6c75e",
+      "-signblockprivatekey=5KbckfYoaBR1a7PuR2JAE2E1jGZ4hMmmCT9nNwJ3rfqf78oWr92"
+    ],
+    "return_code": 1,
+    "error_txt": "Error: Uncompressed Aggregate Public Keys are not supported",
+    "description": "Ensure uncompressed pubkeys are not accepted as aggregate pubkey."
+  },
+  {
+    "exec": "./tapyrus-genesis",
+    "args": [
+      "-dev",
+      "-time=1563342688",
+      "-signblockpubkey=04473757a955a23f75379820f3071abf5b3343b78eb54e52373d06259ffa6c550bd0acc38ff1b182a5de0c795aa7109e8348ca5436d65a267ef134af595ec6c75e",
+      "-signblockprivatekey=L55noj9NZSxyX9E6M2ijiUQSDZeXK8ga5YRmHpeFcERmBnvc7bS3"
+    ],
+    "return_code": 1,
+    "error_txt": "Error: Uncompressed Aggregate Public Keys are not supported",
+    "description": "Ensure uncompressed pubkeys are not accepted as aggregate pubkey in dev mode."
+  },
+  {
+    "exec": "./tapyrus-genesis",
+    "args": [
+      "-signblockpubkey=04b0da749730dc9b4b1f4a14d6902877a92541f5368778853d9c4a0cb7802dcfb25e01fc8fde47c96c98a4f3a8123e33a38a50cf9025cc8c4494a518f991792bb7",
+      "-signblockprivatekey=5KSSJQ7UNfFGwVgpCZDSHm5rVNhMFcFtvWM3zQ8mW4qNDEN7LFd"
+    ],
+    "return_code": 1,
+    "error_txt": "Error: Uncompressed Aggregate Public Keys are not supported",
+    "description": "Ensure uncompressed pubkeys are not accepted as aggregate pubkey in dev mode."
```

Comment:
What is the difference between this case and the one before it?

Reply:
this shouldn't be dev mode. there is no "-dev" parameter. Fixed it. Other than that it's the same test. just different pub key.